frontend/src/fixtures/userCommonsFixtures.js

```diff
@@ -8,7 +8,7 @@ const userCommonsFixtures = {
             },
             "commons":  {
                 "id":1,
-                "name":"Anika's cows",
+                "name":"Table 11",
                 "day": 5,
                 "endDate":"6/11/2021",
                 "totalPlayers": 50,
```

Comment:
Since we now have a new field in the commons from the PR below ( `showLeaderboard`), that should probably be a part of these fixtures now as well.   Please consider adding that to these commons entries.

https://github.com/ucsb-cs156-s22/s22-6pm-happycows/pull/36

Reply:
Added this new material with most recent pr.